sakai-evaluation-impl/src/java/org/sakaiproject/evaluation/logic/externals/EvalExternalLogicImpl.java

```diff
@@ -686,35 +772,97 @@ public int countEvalGroupsForUser(String userId, String permission) {
     }
 
     /* (non-Javadoc)
-     * @see org.sakaiproject.evaluation.logic.externals.ExternalEvalGroups#countUserIdsForEvalGroup(java.lang.String, java.lang.String)
+     * @see org.sakaiproject.evaluation.logic.externals.ExternalEvalGroups#countUserIdsForEvalGroup(java.lang.String, java.lang.String, java.lang.Boolean)
      */
-    public int countUserIdsForEvalGroup(String evalGroupId, String permission) {
-        // get the count from the method which retrieves all the groups,
-        // this method might be better to retire
-        return getUserIdsForEvalGroup(evalGroupId, permission).size();
+    public int countUserIdsForEvalGroup( String evalGroupID, String permission, Boolean sectionAware )
+    {
+        return getUserIdsForEvalGroup( evalGroupID, permission, sectionAware ).size();
```

Comment:
This would be good, however the method that does the work (`getUserIdsForEvalGroup()`) is calling Sakai API services, which don't provide alternative 'count' methods. Mind you, the services themselves probably implement their own caching.

If I were running custom queries behind the scenes here, then it wouldn't be a problem to write another query to get a count. Unfortunately, this is not the case.